README.md

```diff
@@ -24,7 +24,7 @@ SUNRISE comes with some read-only data set by default that you can use. But if y
 Once you have created your commercetools project and populated it with data, follow the next steps to connect to it.
 
 ### 1. Create an API client for a SPA
-In the [Admin Center](https://admin.commercetools.com/), select your project and go to the `Developers` section and then the `API Clients` tab. Now click on the `Add API client` button to display a form. There, enter a descriptive name for your new API client and click on `Select permissions for an API client suited for a mobile or a single page application`, found below the `Permissions` checkboxes.
+In the [Merchant Center](https://mc.commercetools.com/), select your project and go to `Settings` then click on `Developer Settings` section and you should find yourself in the New API Client tab. Now click on the `Create New API client` button to display a form. There, enter a descriptive name for your new API client and click on `Select Template for a Mobile client` then click on `Create API Client` button, now you should be able to see your project credentials.
```

Comment:
"There, enter a descriptive name for your new API client and click on `Select Template for a Mobile client` then click on `Create API Client` button"
To avoid having too many instructions in a single line, I would stop after the template and in another sentence say something like "Once you have filled the form, submit it by clicking on `Create API button`, you should now be able to..."